Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -110,6 +110,25 @@ final class MongoCollectionTests: MongoSwiftTestCase {
         expect(try self.coll.insertOne(self.doc1)).toNot(throwError())
     }
 
+    func testDropWithWriteConcerns() throws {
+        let encoder = BSONEncoder()
+
+        let center = NotificationCenter.default
+
+        let writeConcern = try WriteConcern(journal: true, w: .number(1))
+        let opts = DropCollectionOptions(writeConcern: writeConcern)
+
+        let observer = center.addObserver(forName: nil, object: nil, queue: nil) { notif in
+            print(notif)
```

Comment:
rather than just printing the notifications, we should inspect them and make sure the commands being sent include the write concern documents.
Specifically we should look for the [`CommandStartedEvent`](https://github.com/mongodb/mongo-swift-driver/blob/cbfa18511567971a86c853b9e0c2ce20cc1bf18f/Sources/MongoSwift/APM.swift#L23)s, and we should make sure that their `command` fields look how we expect. 

The command document might contain some extra info that we don't care about so probably we can just make sure the command name is "drop" and that there is "writeConcern" field that matches the write concern you're sending in the options.

Reply:
let's remove this `print` statement now.
one other thing I would suggest is tracking somehow that we actually saw a `CommandStartedEvent` happen. 
consider if for some reason your call to `collection.drop` below didn't do anything - we would never execute any of the code within this block but the test would still pass.